src/main/java/net/snowflake/client/jdbc/SnowflakeSQLLoggedException.java

```diff
@@ -261,6 +272,20 @@ public SnowflakeSQLLoggedException(
     sendTelemetryData(null, SQLState, vendorCode, session, this);
   }
 
+  public SnowflakeSQLLoggedException(
+      String queryId,
+      SFBaseSession session,
+      String SQLState,
+      int vendorCode,
+      Throwable ex,
+      Object... params) {
+    super(queryId, ex, SQLState, vendorCode, params);
+    // add telemetry
```

Comment:
Telemetry added as I can see?

Reply:
it's a comment in some places here - I can clean it up